todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -68,13 +89,60 @@ public static TaskDetailFragment newInstance(String taskId) {
     @Override
     public void onResume() {
         super.onResume();
-        mPresenter.subscribe();
+        bind();
     }
 
     @Override
     public void onPause() {
+        unbind();
         super.onPause();
-        mPresenter.unsubscribe();
+    }
+
+    private void bind() {
+        mSubscription = new CompositeSubscription();
```

Comment:
I'm not 100% sure of what you mean with binding declaration and binding implementation? 
I don't think that having pieces of code like this:
```java
        mSubscription.add(getViewModel().getLoadingIndicator()
                .subscribeOn(Schedulers.computation())
                .observeOn(AndroidSchedulers.mainThread())
                .subscribe(new Subscriber<Boolean>() {
                    @Override
                    public void onCompleted() {
                    }

                    @Override
                    public void onError(Throwable e) {
                        showMissingTask();
                    }

                    @Override
                    public void onNext(Boolean active) {
                        setLoadingIndicator(active);
                    }
                }));
```
For every View is so much. The View decides what it's interested in (for example the loading), and where to display the result of the loading. 
Where would you put the `subscribeOn` and `observeOn`?